code/modules/vore/fluffstuff/custom_clothes_vr.dm

```diff
@@ -676,6 +679,9 @@
 		else
 			return 1
 
+/obj/item/clothing/under/fluff/screesuit/digest_act(var/datum/belly/belly = null)
+	return FALSE
```

Comment:
oh lol the screesuit's still in? I thought I removed it entirely when markings became a thing. Musta just removed it from the fluff items .txt